src/page/News.tsx

```diff
@@ -0,0 +1,38 @@
+import { remoteData } from 'cbioportal-frontend-commons';
+import { observer } from 'mobx-react';
+import ReactMarkdown from 'react-markdown';
+import rehypeRaw from 'rehype-raw';
+import rehypeSanitize from 'rehype-sanitize';
+import remarkGfm from 'remark-gfm';
+import * as React from 'react';
+import './News.scss';
+import $ from 'jquery';
+
+@observer
+class News extends React.Component<{}> {
+
+    readonly source = remoteData<string>(async () => {
+        return await $.get("https://raw.githubusercontent.com/leexgh/genome-nexus-frontend/news-page/src/doc/News.md");
```

Comment:
Will change to `https://raw.githubusercontent.com/genome-nexus/genome-nexus-frontend/master/src/doc/News.md` after review